fe/fe-core/src/main/java/com/starrocks/http/rest/BootstrapFinishAction.java

```diff
@@ -87,20 +91,24 @@ public void execute(BaseRequest request, BaseResponse response) throws DdlExcept
                     clusterId = Integer.parseInt(clusterIdStr);
                 } catch (NumberFormatException e) {
                     result.status = ActionStatus.FAILED;
-                    result.msg = "invalid cluster id format: " + clusterIdStr;
+                    LOG.info("invalid cluster id format: {}", clusterIdStr);
+                    result.msg = "invalid parameter";
                 }
 
                 if (result.status == ActionStatus.OK) {
                     if (clusterId != GlobalStateMgr.getCurrentState().getNodeMgr().getClusterId()) {
                         result.status = ActionStatus.FAILED;
-                        result.msg = "invalid cluster id: " + GlobalStateMgr.getCurrentState().getNodeMgr().getClusterId();
+                        LOG.info("invalid cluster id: {}",
+                                GlobalStateMgr.getCurrentState().getNodeMgr().getClusterId());
+                        result.msg = "invalid parameter";
                     }
                 }
 
                 if (result.status == ActionStatus.OK) {
                     if (!token.equals(GlobalStateMgr.getCurrentState().getNodeMgr().getToken())) {
                         result.status = ActionStatus.FAILED;
-                        result.msg = "invalid token: " + GlobalStateMgr.getCurrentState().getNodeMgr().getToken();
+                        LOG.info("invalid token: {}", GlobalStateMgr.getCurrentState().getNodeMgr().getToken());
```

Comment:
similiar, it is the `token` invalid, not the `GlobalStateMgr....getToken()` invalid.

Reply:
fixed